components/selection/selection-mixin.js

```diff
@@ -194,7 +193,7 @@ export const SelectionMixin = superclass => class extends RtlMixin(superclass) {
 
 		this._updateObserversRequested = true;
 		setTimeout(() => {
-			const info = this.getSelectionInfo();
+			const info = this.getSelectionInfo(true);
```

Comment:
This is required so that providers will "roll-up" all the keys for the entire selection tree for observers.